test/parallel/test-http-server-keep-alive-timeout.js

```diff
@@ -1,77 +1,122 @@
 'use strict';
 
-const common = require('../common');
-const assert = require('assert');
-const http = require('http');
-const net = require('net');
+module.exports.testServerKeepAliveTimeout = function(common, isHttps) {
```

Comment:
This is a test file, not a library file. It shouldn't be exporting. -100 to exporting something to another test file from it.